dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/CommandLine.java

```diff
@@ -78,6 +78,10 @@ static ArgumentParser getArgumentParser() {
                 .help("regex of repository names to exclude from pull request generation");
         parser.addArgument("-B")
                 .help("additional body text to include in pull requests");
+        parser.addArgument("-s", "--" + Constants.SKIP_PR_CREATION)
```

Comment:
Would `--dry-run` make sense?

Reply:
Actually, this seems to be of a different intent than just skipping PR creation as well. Should this be a different command altogether?

Reply:
What would you suggest the subcommand name be @justinharringa ? 

dry-run isn't good because we intend to change our datastore, which is a mutation. 

Naming is always hard, but we can also change this flag to be `--only-persist-tag-store`.

I'm honestly partial to keeping what @avimanyum has right now.

Reply:
Perhaps a mode called `update-store`? Seems that would also be a simple change.

Reply:
To keep this consistent between `child` and `parent` subcommands, we can add a similar check in the `child` sub command execution as well.